docs/fixes/2026-07-21-terraform-lint-auth-disabled-yaml-functions.md

```
@@ -0,0 +1,75 @@
+
+CI's `[lint] quick-start-advanced` job ran:
+
+```shell
+atmos terraform lint sns-topic -s plat-ue2-dev --identity=false
+```
+
+with a comment explaining the intent: "Lint reads static HCL only. Disable the quick-start
+stack's emulator identity because this CI job intentionally does not start the emulator." The
+job failed anyway:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)